packages/tsconfig-reference/copy/pt/options/preserveConstEnums.md

```diff
@@ -0,0 +1,56 @@
+---
+display: "Preserve Const Enums"
```

Comment:
Preservar Enums Constantes

Reply:
Changed "Preserve" to "Mantenha". What do u think?